tests/fixtures/definitions/without-id.bp

```diff
@@ -0,0 +1,8 @@
+models:
+  Name:
+    softDeletes
```

Comment:
How does this test `id: false`?

Reply:
Ah yes. I copied the original test and stub over to the new test and needed some attributes to remain. Figured I’d only delete the ID attribute.

Reply:
I guess I don't understand what this test proves. I thought you were trying to support `id: false` disabling the id field. As such, I'd expect this definition to have `id: false` or the test to fail...

Reply:
Omitting `id` should not change the current behavior. Explicitly setting`id: false` should.

Reply:
Exactly. That's why I expect this fixture to have `id: false` based on the test.

Reply:
So sorry. I’ll fix this!

Reply:
I explicitly set `id: false` and updated the test to prove `'id' => false` is set.